addons/website_sale/tests/test_delivery_ui.py

```diff
@@ -61,6 +55,14 @@ def setUpClass(cls):
     def test_01_free_delivery_when_exceed_threshold(self):
         if self.env['ir.module.module']._get('payment_custom').state != 'installed':
             self.skipTest("Transfer provider is not installed")
+
+        transfer_provider = self.env.ref('payment.payment_provider_transfer')
+        transfer_provider.write({
+            'state': 'enabled',
+            'is_published': True,
+        })
```

Comment:
Since you are fixing it we can make it cleaner :)
```suggestion
        transfer_provider.write({'state': 'enabled', 'is_published': True})
```